gradle/wrapper/gradle-wrapper.properties

```diff
@@ -18,4 +18,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
```

Comment:
all not bin?

Reply:
sigh.   copied & automated from main repo without looking carefully enough.  will same with the other 20 PRs I just opened).